cityofaustin/atd-vz-data
# Implement custom 404 page

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/12995

Even though the issue only specifies to do this for crash ids that dont exist, i also implemented it for location ids.

I know the mockup doesn't include the navbar on the left side, but I thought that leaving the navbar would make it easier for users to navigate to any other tab if they reach the 404 page rather than only having the option to go back to the Home page. Let me know what y'all think.

## Testing
**URL to test:** 
[Netlify](https://deploy-preview-1291--atd-vze-staging.netlify.app/#/dashboard)

**Steps to test:**
1. Go to the crashes page and then type in an int after /crashes/ in the URL that isn't a crash id we have, like https://deploy-preview-1291--atd-vze-staging.netlify.app/#/crashes/22. The 404 page should be rendered.
2. Click on the Back to Homepage button to make sure it works.
3. Go to the Locations page and do the same thing, for example go to https://deploy-preview-1291--atd-vze-staging.netlify.app/#/locations/h750. You should once again get the 404 page if that location id doesn't exist.
4. Typing in a random url like https://deploy-preview-1291--atd-vze-staging.netlify.app/#/hello should just reroute you back to the dashboard which I feel like is acceptable behavior.

---
#### Ship list
